packages/coinstac-ui/app/render/styles/alert.scss

```diff
@@ -17,3 +17,15 @@
   }
 }
 
+/**
+ * Notifications.
+ *
+ * Override react-notification-system-redux component styles.
+ */
+
+.notification {
+  border-radius: 0 !important;
+  border: none !important;
+  opacity: 0.75 !important;
+  box-shadow: none !important;
+}
```

Comment:
This works, but the notification looks a little jerky on my machine. Might have to do with recently having built and the thing being a little overworked. Anyway, what doesn't cause it to be a little jerky, and will also allow us to avoid the dreaded `!important`, is using the component's [built-in way for overriding styles](https://github.com/gor181/react-notification-system-redux#usage).

In `coinstac-ui/app/render/components/app.jsx` add the following to the top just above the class declaration:
```js
const styles = {
  notifications: {
    NotificationItem: {
      DefaultStyle: {
        borderRadius: 0,
        border: 'none',
        opacity: 0.75,
        boxShadow: 'none',
        fontWeight: 'bold',
      },
    }
  }
};
```

Then change the *Notifications* component around line 49 to be something like:

```html
<Notifications
  notifications={notifications}
  style={styles.notifications}
/>
```

This will allow you to get rid of the scss here and, according to the docs, modify the styles of the various notification types should you want to some day.

Reply:
Done :)